### ISSUE TITLE
Set _NET_WM_STATE_HIDDEN on other windows when one is fullscreened

### ISSUE THREAD

**Issue by PMunch** [avahe-kellenberger/nimdow#189]
I use the Picom compositor and it has a rule for making the background of my terminals transparent. This works well, but when I fullscreen a terminal in Nimdow I can see the other windows below, which is very disturbing when trying to read text. In order to avoid this in i3 I have a rule that `_NET_WM_STATE = _NET_WM_STATE_HIDDEN` should be completely transparent. Apparently i3 then either removes those windows or just sets that hint and lets my compositor handle the hiding.

So in summary I believe that `_NET_WM_STATE_HIDDEN` should be set on windows when another window on the same screen is fullscreened.
